nautobot/core/management/commands/generate_test_data.py

```diff
@@ -374,6 +375,8 @@ def handle(self, *args, **options):
                 "flush", "--no-input", "--database", options["database"], inhibit_post_migrate=inhibit_post_migrate
             )
 
+        cache.delete("nautobot.extras.utils.change_logged_models_queryset")
```

Comment:
Interesting... if this is necessary/relevant, let's add a comment and change fragment about it. Are there any other content-type querysets that have a similar cache we'd need to clear?

Reply:
I was seeing this when I was running `generate_test_data --flush` on the default db. But I can't reproduce it now so my database may have been in a bad state. Going to revert it for now

```
Creating 100 object changes...
Got a ValidationError for extras | object metadata  deleted by bryanlam: {'changed_object_type': ['content type instance with id 156 does not exist.']}
Traceback (most recent call last):
```

Reply:
I ran into this again and I think we should track this in a follow-up issue. We probably need to invalidate the entire cache when we do a database flush. If the nautobot server is running when you run this command, any cached database data will become invalid when the database is flushed.

Reply:
Yeah, probably doing a full-on `cache.clear()` would work well enough for now.